File: lab03_stack/lab03_task02/scripts/lab03_task02_sub_node.py
#!/usr/bin/env python

import rospy
import numpy as np
##TODO: include the library for a msg for an array of doulbe
##TODO: Complete the callback function. Your callback function should rotate the input point and print the result out on the screen.
## Show the computational time on the screen.

def subscribeCallback(msg):
    print  'test'


def rot_converter():

    rospy.init_node('rotate_point_subscriber', anonymous=True)
    ##TODO: Define your subscriber and link this with the callback function.
    rate = rospy.Rate(10)

    rospy.spin()
    rate.sleep()

if __name__ == '__main__':
    try:
        rot_converter()
    except rospy.ROSInterruptException:
        pass

File: lab03/lab03_task02/src/lab03_task02_sub_node.cpp
#include "ros/ros.h"
//TODO: include the library for a msg for an array of doulbe

//TODO: Complete the callback function. Your callback function should rotate the input point and print the result out on the screen. Show the computational time on the screen.
void subscribeCallback();

int main(int argc, char **argv)
{
    ros::init(argc, argv, "rotate_point_subscriber");

    //TODO: Define your subscriber and link this with the callback function.

    ros::spin();

    return 0;
}
